### ISSUE TITLE
Since 5.4.0 --exclude does not work anymore

### ISSUE THREAD

**Issue by Martin-Wegner** [palantir/tslint#2852]
### Bug Report

- __TSLint version__: 5.4.0
- __TypeScript version__: 2.3.4
- __Running TSLint via__: CLI

Since 5.4.0 --exclude does not work anymore and breaks my build :(

`tslint --config ./tslint.json --exclude '**/server-api/**' --project ./tsconfig.json --type-check`

The `server-api` directory contains generated code which cannot be linted and therefore must be excluded.

```
|-tslint.json
|-tsconfig.json
|-src
  |-server-api
    |-...
  |-app
    |-...
|-...
```

Until 5.3.2 this worked like a charme...

**Comment by sandersn** [comment]
This doesn't seem to be fixed in 5.4.1. Typescript's build is still failing, for example:

https://travis-ci.org/Microsoft/TypeScript/jobs/238410879

You can repro this locally with `gulp lint` after cloning Microsft/TypeScript and running `npm install`. Our command is as follows:

```sh
node node_modules/tslint/bin/tslint Gulpfile.ts 'scripts/tslint/*.ts' 'src/**/*.ts' --exclude src/lib/es5.d.ts --exclude 'src/lib/*.generated.d.ts' --format stylish
```

**Comment by Martin-Wegner** [comment]
For me 5.4.1 fixed the problem...

**Comment by jmquigley** [comment]
It seems that using multiple "--exclude" options on one CLI are not working (at least not for me).  I have a module build that is failing.  I think that it is only respecting the last one from the CLI.  According to the options one should be allowed to do this: "This option can be supplied multiple times if you need multiple globs to indicate which files to exclude."

**Comment by sandersn** [comment]
Yes, I filed a separate issue in #2855 that tracks exactly this.
